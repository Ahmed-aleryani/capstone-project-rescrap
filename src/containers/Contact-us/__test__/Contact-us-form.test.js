import React from 'react';
import renderer from 'react-test-renderer';
import ContactUsForm from '../Contact-us-form';

it("matches the snapshot", () =>{
    const tree = renderer.create(<ContactUsForm/>).toJSON();
    expect(tree).toMatchSnapshot();
})