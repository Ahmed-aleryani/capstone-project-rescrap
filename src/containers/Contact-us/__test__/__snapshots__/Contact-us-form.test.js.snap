// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
<div
  className="sm:h-full lg:h-screen w-full px-20"
>
  <form
    className="bg-white px-4 pb-16 pt-5 "
  >
    <h1
      className="text-3xl normal text-green-100 pb-1"
    >
      Let's Contact!!
      <h1 />
    </h1>
    <div
      className="py-1"
    >
      <label
        className="text-sm"
      >
        Name
      </label>
      <input
        className="shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline\\" id=\\"username"
        placeholder="Name"
      />
    </div>
    <div
      className="py-1"
    >
      <label
        className="text-sm"
      >
        Subject
      </label>
      <input
        className="shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline\\" id=\\"username"
        placeholder="Subject"
      />
    </div>
    <div
      className="py-1"
    >
      <label
        className="text-sm"
      >
        Email
      </label>
      <input
        className="shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline\\" id=\\"username"
        placeholder="Email"
      />
    </div>
    <div
      className="py-1"
    >
      <label
        className="text-sm"
      >
        Message
      </label>
      <textarea
        className="shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline\\" id=\\"username"
        placeholder="Message"
      />
    </div>
    <div
      className=" text-center py-1"
    >
      <button
        className="bg-green-100 py-4 w-4/5 border rounded text-white "
        type="submit"
      >
        Submit
      </button>
    </div>
  </form>
</div>
`;
